src/lib.rs

```diff
@@ -92,11 +92,57 @@ impl<P: Pipe> PythonParserBuilder<P> {
 
     pub fn load_and_init_class<'p>(py: Python<'p>, module_name: &str, class_name: &str, options: &[(String, String)]) -> PyResult<PyObject> {
         let module = try!(Self::load_module(py, module_name));
+        let dict = module.dict(py);
+
+        if dict.contains(py, "error").unwrap() {
```

Comment:
please use `try!` instead of `unwrap()`:

```
        if try!(dict.contains(py, "info")) {
            warn!("Already implemented info function, omitting callback definition.");
        } else {
            try!(dict.set_item(py, "info", py_fn!(python_info_callback(info_message: &str))));
        }
```